src/components/body/slideshow/index.js

```diff
@@ -62,19 +88,25 @@ const duration = 300
 const defaultCurIndex = 0
 
 const SlideshowFlexBox = styled.div`
-  max-width: ${mockup.desktop.container.width}px;
   width: 100%;
   display: flex;
   justify-content: space-between;
   flex-wrap: wrap;
 
+  ${mq.tabletAndBelow`
+    // 10px is the border-left of body
+    margin-left: -10px;
+    // 20px is the border-(right|left) of body
+    width: calc(100% + 20px);
```

Comment:
The `width` of `Multimedia` in `src/components/body/styled.js` on mobile and tablet need to be updated as `width: calc(100% + 20px);` also.